README.md

```diff
@@ -15,6 +15,10 @@
 [![YouTube Channel](https://img.shields.io/badge/YouTube-Subscribe-red)](https://www.youtube.com/channel/UCa57PMqmz_j0wnteRa9nCaw)
 <br><br><br><br>
 
+#### *Read this in [other languages](translations/TRANSLATIONS.md).*
```

Comment:
Do we have translations.md? Maybe a good model is:

*Read this in other languages : [KR](translations/README-ko.md) [FR](translations/README-fr.md) [JA](translations/README-ja.md)*

Reply:
Ok, I see this is from https://github.com/openebs/openebs/blob/master/translations/TRANSLATIONS.md -- which is also fine.  We can create this translations

Reply:
You mean without flag?

Reply:
I missed this file pushed it, pushed the same. Also directly clicking on the flag emoji it would go there. Followed the same approach as above.